packages/in_app_purchase/in_app_purchase_storekit/darwin/Classes/InAppPurchasePlugin.swift

```diff
@@ -116,10 +120,11 @@ public class InAppPurchasePlugin: NSObject, FlutterPlugin, InAppPurchaseAPI {
   public func storefrontWithError(_ error: AutoreleasingUnsafeMutablePointer<FlutterError?>)
     -> SKStorefrontMessage?
   {
-    if #available(iOS 13.0, *), let storefront = getPaymentQueueHandler().storefront {
-      return FIAObjectTranslator.convertStorefront(toPigeon: storefront)
+    if #available(iOS 13.0, *) {
+      return FIAObjectTranslator.convertStorefront(toPigeon: getPaymentQueueHandler().storefront)
+    } else {
+      fatalError()
```

Comment:
need message here. Also why the behavior change from returning nil to fatalError

Reply:
oopsies. reverted.